installing/installing_ocp/install-secured-cluster-config-options-ocp.adoc

```diff
@@ -0,0 +1,12 @@
+:_content-type: ASSEMBLY
+[id="install-secured-cluster-config-options-ocp"]
+= Optional - Configuring Secured cluster configuration options for RHACS using the Operator
+:context: install-secured-cluster-config-options-ocp
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+[role="_abstract"]
+This topic provides information about optional configuration options that you can configure using the Operator.
```

Comment:
Suggestion:

"optional configuration options that you can configure" reads a little awkward to me, how about the following:

"This topic provides information about optional configurations that you can make using the Operator."

Reply:
👍 or another word like "optional configuration settings", etc.